github-sync/github-data/sigstore/users.yaml

```diff
@@ -211,6 +211,8 @@ users:
         role: member
       - name: fulcio-codeowners
         role: member
+      - name: model-transparency-codeowners
```

Comment:
youll have to add the membership in a later PR, otherwise pulumi fails cause sometimes it runs the user sync first

Reply:
just add the new users without any memberships first

Reply:
I'll push the other commit to a separate PR then and have this one only create the team.